Google.Api.Generator/Generation/ServiceAbstractClientClassCodeGenerator.cs

```diff
@@ -82,7 +85,22 @@ private ClassDeclarationSyntax Generate()
 
         private PropertyDeclarationSyntax ChannelPool(PropertyDeclarationSyntax defaultScopes) =>
```

Comment:
Maybe have ChannelPool method receive `PropertyDeclarationSyntax useJwtAccessWithScopes` the same as it receives the `defaultScopes`. Just for simetry. And then just swat the order of lines 55/56.

Reply:
Can do. I'll do the same with other things where I'm just creating extra declarations for the heck of it :)